internal/cli/capact/capact.go

```diff
@@ -13,8 +13,13 @@ const (
 	LocalDockerTag = "dev"
 	// LocalDockerPath path used when building local images
 	LocalDockerPath = "local"
+
 	// KindEnv default name for kind environment
 	KindEnv = "kind"
+	// GkeEnv default name for GKE environment
+	GkeEnv = "gke"
+	// EksEnv default name for EKS environment
+	EksEnv = "eks"
```

Comment:
```suggestion
	// GKEEnv default name for GKE environment
	GKEEnv = "gke"
	// EKSEnv default name for EKS environment
	EKSEnv = "eks"
```
respectively the rest code needs to be updated